cpp-ch/local-engine/Parser/SerializedPlanParser.h

```diff
@@ -59,10 +59,10 @@ static const std::map<std::string, std::string> SCALAR_FUNCTIONS
        {"alias", "alias"},
 
        /// datetime functions
-       {"get_timestamp", "parseDateTimeInJodaSyntaxOrNull"}, // for spark function: to_date/to_timestamp
+       {"get_timestamp", "sparkParseDateTimeInJodaSyntaxOrNull"}, // for spark function: to_date/to_timestamp
```

Comment:
Copying all the codes of original function is definitely a ugly but last possible way. What's the difference between parseDateTimeInJodaSyntaxOrNull and sparkParseDateTimeInJodaSyntaxOrNull. It is possible to contribute to ClickHouse first?